examples/bind-shopping.amp.html

```diff
@@ -0,0 +1,454 @@
+<!doctype html>
```

Comment:
> Good point! Though for amp-live-list, we'll only have to scan the updated children rather than rescanning the entire DOM. Assuming a modest number of updates, do you think this is still a problem?

Not really a problem. But important to acknowledge that we'd create a problem that we wouldn't have with a custom element. It's a tradeoff, so we need to make sure we get our +/- right.